types/api/v1beta1/event.pb.go

```diff
@@ -573,11 +573,12 @@ type Thread struct {
 
 	Start          *timestamp.Timestamp  `protobuf:"bytes,1,opt,name=start,proto3" json:"start,omitempty"`
 	Name           string                `protobuf:"bytes,2,opt,name=name,proto3" json:"name,omitempty"`
-	Tid            *wrappers.UInt32Value `protobuf:"bytes,3,opt,name=tid,proto3" json:"tid,omitempty"`
-	NamespacedTid  *wrappers.UInt32Value `protobuf:"bytes,4,opt,name=namespaced_tid,json=namespacedTid,proto3" json:"namespaced_tid,omitempty"`
-	Syscall        string                `protobuf:"bytes,5,opt,name=syscall,proto3" json:"syscall,omitempty"`
-	Compat         bool                  `protobuf:"varint,6,opt,name=compat,proto3" json:"compat,omitempty"`
-	UserStackTrace *UserStackTrace       `protobuf:"bytes,7,opt,name=user_stack_trace,json=userStackTrace,proto3,oneof" json:"user_stack_trace,omitempty"`
+	EntityId       *wrappers.UInt32Value `protobuf:"bytes,3,opt,name=entity_id,json=entityId,proto3" json:"entity_id,omitempty"`
```

Comment:
Did you edit this file by hand ? This file is automatically generated from the proto file.

Reply:
Oh, didn't know this.
I will revert the change

Reply:
What file is the source of this?